compiler/src/iree/compiler/Codegen/Common/LinkTuningSpecsPass.cpp

```diff
@@ -61,6 +62,210 @@ static bool consumesInputOp(NamedSequenceOp op) {
   return false;
 }
 
+static bool hasConsumedArgument(ForeachMatchOp op) {
+  Value operand = op->getOperand(0);
+  if (auto blockArg = dyn_cast<BlockArgument>(operand)) {
+    Operation *parentOp = blockArg.getOwner()->getParentOp();
+    if (auto namedSequenceOp = dyn_cast<NamedSequenceOp>(parentOp)) {
+      return namedSequenceOp.getArgAttr(blockArg.getArgNumber(),
+                                        kArgConsumedAttrName) != nullptr;
+    }
+  }
+  return false;
+}
+
+struct TuningSpecsToMerge {
+  // - `namedSequenceOpsToMove`: Contains `NamedSequenceOp`s that either:
+  //     - Are not named `__kernel_config`.
+  //     - Do not have the `iree_codegen.tuning_spec_entrypoint` attribute.
+  SmallVector<NamedSequenceOp> namedSequenceOpsToMove;
```

Comment:
The format is weird -- why repeat the field name and make it a bullet point?
```suggestion
  // Contains `NamedSequenceOp`s that either:
  //  - Are not named `__kernel_config`.
  //  - Do not have the `iree_codegen.tuning_spec_entrypoint` attribute.
  SmallVector<NamedSequenceOp> namedSequenceOpsToMove;
```